itm_dashboard/dashboard-ui/src/components/Results/alignmentScore.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+
+const AlignmentScoreBox = ({ performer, data, scenario }) => {
+  let score = "Not Available"
+  let sessionId = null;
+
+  if(data.history !== null && data.history !== undefined) {
+    if( data.history[data.history.length - 1].response.score !== undefined) {
+        score = data.history[data.history.length - 1].response.score.toFixed(4);
+    }
+
+    for(let i=0; i < data.history.length - 1; i++) {
+      if(data.history[i].command === "TA1 Alignment Target Session ID") {
+        sessionId = data.history[i].response; 
+      }
+    }
+  }
+
+  return (
+    <Box className="scoreBox">
+      <Typography variant="h5">
+        {performer}
+        <br />
+      </Typography>
+      <Typography variant="h5">
+        Alignment Score: {score}
+        {scenario === 'kickoff-demo-scenario-1' && 
```

Comment:
i believe == will convert type as necessary before checking for equality. could prevent future bug if scenario is ever not of type raw string

Reply:
@dereknop - it would do a comparison on any type, however if scenario isn't a string, then this would be a bug and this should fail.  The bug would be if scenario wasn't a type of string.